Makefile

```diff
@@ -36,10 +36,20 @@ TESTFLAGS    :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# Static linking with glibc is a bad time; see
+# https://github.com/golang/go/issues/13470.
+# If a static build is requested, assume musl is installed (it is in
+# the cockroachdb/builder docker container) and link against it
+# instead.
+CC = /usr/local/musl/bin/musl-gcc
```

Comment:
Well, it works just fine with musl. There's an ongoing discussion between @bdarnell and I in an "outdated diff" - basically, I think it's worth keeping this because someone may want a static build, and we've already done all the work to figure out how to make that safe.